lib/Digitick/Sepa/TransferFile/Facade/CustomerCreditFacade.php

```diff
@@ -61,8 +61,15 @@ public function addTransfer($paymentName, array $transferInformation)
             $transferInformation['creditorName']
         );
         $transfer->setBic($transferInformation['creditorBic']);
-        $transfer->setRemittanceInformation($transferInformation['remittanceInformation']);
-
+        
+        if(array_key_exists('creditorReferenceInformation', $transferInformation) &&
+            strlen($transferInformation['creditorReferenceInformation'])>5 ) {
```

Comment:
What is this constraint (>5) for  ? and if really usefull can you create a constant from ?